lib/assessment.js

```diff
@@ -5,7 +5,14 @@
 // and returns the minimum from that set
 // assume that all the argument are numbers
 var min = function min() {
-
+  var sorted = [];
+  for (var i = 0; i < arguments.length; i++) {
+    sorted[i] = arguments[i];
+  }
+    sorted.sort(function(a,b){
+      return a - b;
+  });
+    return sorted[0];
```

Comment:
Clever! This is a novel way of getting the solution.
Assuming that you didn't have `sort` at your disposal, though, you might have to revert to doing things the hard way, e.g. using a `for` loop.